app/models/spree/adjustment_decorator.rb

```diff
@@ -7,4 +7,11 @@ def avatax_cache_key
     key << self.amount
     key.join('-')
   end
+
+  def tax_breakdown
+    meta[:tax_breakdown] unless meta[:tax_breakdown].nil?
```

Comment:
Shouldn't this be `return meta[:tax_breakdown] unless meta[:tax_breakdown].nil?`

Reply:
of course, nice catch